components-registry-service-server/src/main/kotlin/org/octopusden/octopus/components/registry/server/service/impl/ComponentManagementServiceImpl.kt

```
@@ -4114,6 +4118,302 @@ class ComponentManagementServiceImpl(
                 },
         )
 
+    // ── SYS-066: one-off composite field-override split ──────────────────────
+
+    @Suppress("LongMethod", "NestedBlockDepth", "CyclomaticComplexMethod")
+    override fun splitCompositeFieldOverrides(
```

detekt (1.23.8)

[warning] The function splitCompositeFieldOverrides appears to be too complex based on Cognitive Complexity (complexity: 40). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)